pages/_app.tsx

```diff
@@ -49,6 +51,27 @@ function CustomApp({
 }: AppProps & AppOwnProps) {
     const [queryClient] = useState(() => new QueryClient(queryClientConfig));
 
+    const [themeMode, setThemeMode] = useState<EuiThemeColorMode>(
```

Comment:
The _app is getting a bit bulky. Think you could move all this color-mode stuff in a separate provider.